addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -438,6 +439,37 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
         }
         this._addHiddenField(value, fieldName);
     },
+    /**
+     * Prompts the user to save changes before being redirected
+     * towards an action specified in value
+     */
+    onClickCreateButton: function (previewMode, value, params) {
```

Comment:
- Add a point at the end of your doc sentence.
- Add `@see this.selectClass for parameters` like we did everywhere.
- Your option method should not start with "on" and indicate when it's called but *what* it is doing
- Name value, widgetValue like everywhere else

Reply:
I'm confused about these two:
- Add @see this.selectClass for parameters like we did everywhere.
- Name value, widgetValue like everywhere else

All of the method next to this one use `previewMode, value, params`, and none of them have `@see this.selectClass for parameters`. Why would this one be different?

Reply:
Because the other ones of this file are also not following the convention, but that's not a reason to introduce even more badly shaped definitions.